thirdeye-server/src/main/java/ai/startree/thirdeye/core/ConfusionMatrix.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.core;
+
+public class ConfusionMatrix {
```

Comment:
MINOR. For metrics classes I've often seen implementations that are more fluent-api-like, and that requires less knowledge by the calling class. 

Eg
ConfusionMatrix
public addTruePositive()
public addTrueNegative()
public addFalsePositive()
public addFalseNegative()
public addUnclassified()
+ public addXXX(int n) to add many observations at the same time

and the ConfusionMatrix maintains the count internally. AppAnalyticsService does not have to know how to count.

Reply:
Got it. Created methods to `inc`, `dec` and `add` to the variables.